src/esn.calendar.libs/app/components/event/form/event-form.controller.js

```diff
@@ -288,6 +288,38 @@ function CalEventFormController(
     });
   }
 
+  function _changeOrganizerParticipation(status) {
+    const eventPayload = $scope.event.clone();
+    const currentlyEditedEvent = $scope.editedEvent.clone();
+
+    eventPayload.setOrganizerPartStat(status);
+
+    calEventService.changeParticipation(eventPayload.path, eventPayload, session.user.emails, status, eventPayload.etag)
+      .then(response => {
+        if (!response) {
```

Comment:
When `response` is undefined? There is no action on this?

Reply:
added a failure notification.